sentry-kotlin-extensions/src/main/java/io/sentry/kotlin/SentryCoroutineExceptionHandler.kt

```diff
@@ -0,0 +1,29 @@
+package io.sentry.kotlin
+
+import io.sentry.HubAdapter
+import io.sentry.IHub
+import io.sentry.SentryEvent
+import io.sentry.exception.ExceptionMechanismException
+import io.sentry.protocol.Mechanism
+import kotlinx.coroutines.CoroutineExceptionHandler
+import org.jetbrains.annotations.ApiStatus
+import kotlin.coroutines.AbstractCoroutineContextElement
+import kotlin.coroutines.CoroutineContext
+
+/**
+ * Sentry exception handler element for [CoroutineExceptionHandler].
+ */
+@ApiStatus.Experimental
+public class SentryCoroutineExceptionHandler(private val hub: IHub = HubAdapter.getInstance()) : AbstractCoroutineContextElement(CoroutineExceptionHandler), CoroutineExceptionHandler {
+    public constructor() : this(HubAdapter.getInstance())
+
+    override fun handleException(context: CoroutineContext, exception: Throwable) {
```

Comment:
I'm wondering if should check `if (context instanceof SentryContext)` and use the hub from there?

Reply:
`HubAdapter` always forwards to `Sentry` static class, it'd not make any difference.